module-apps/application-call/widgets/StateIcons.cpp

```diff
@@ -19,7 +19,7 @@ namespace gui
         constexpr auto speakerImg    = "call_speaker";
         constexpr auto speakerOnImg  = "call_speaker_on";
         constexpr auto speakerStr    = "app_call_speaker";
-        constexpr auto speakerOnStr  = "app_call_speaker_on";
+        constexpr auto speakerOnStr  = "app_call_speaker";
```

Comment:
Using the same SPEAKER (recommendation from design). SPEAKER ON string will not fit to the box